plugins/.snapshots/TestGenerateSourcePluginDocs-JSON-__tables.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "name": "test_table",
+    "description": "Description for test table",
+    "columns": [
+      {
+        "name": "_cq_source_name",
+        "type": "TypeString"
+      },
+      {
+        "name": "_cq_sync_time",
+        "type": "TypeTimestamp"
+      },
+      {
+        "name": "_cq_id",
+        "type": "TypeUUID"
+      },
+      {
+        "name": "_cq_parent_id",
+        "type": "TypeUUID"
+      },
+      {
+        "name": "int_col",
+        "type": "TypeInt"
+      },
+      {
+        "name": "id_col",
+        "type": "TypeInt",
+        "is_primary_key": true
+      },
+      {
+        "name": "id_col2",
+        "type": "TypeInt",
+        "is_primary_key": true
+      }
+    ],
+    "relations": [
```

Comment:
Sorry for jumping late to this, but can we also have the relations at the top level? Might be easier to process it that way. For example if I was to get a list of all tables, I'll need to recurse into the JSON at the moment.

We can add a another property `parent` to link to the parent table, and keep it `nil` for top level ones, WDYT?

Reply:
Yeah, I was in two minds about it 🤔  I think having it flattened will be good for some use cases, and not-so-good for others. I'm fine with changing it to flattened though. @yevgenypats WDYT?

Reply:
Maybe we can have both?

Reply:
![why not both](https://media.tenor.com/QmMiDHANnBUAAAAC/why-dont-we-have-both-both.gif)